src/scripts/main.js

```diff
@@ -1,3 +1,60 @@
 'use strict';
 
-// write code here
+const tableBody = document.querySelector('.field tbody');
+const appendRow = document.querySelector('.append-row');
+const appendCol = document.querySelector('.append-column');
+const removeRow = document.querySelector('.remove-row');
+const removeCol = document.querySelector('.remove-column');
+const col = document.createElement('td');
+const row = document.createElement('tr');
+const maxVal = 10;
+const minVal = 2;
+
+appendRow.addEventListener('click', () => {
+  const newRow = row.cloneNode(true);
+
+  for (let i = 0; i < tableBody.firstElementChild.children.length; i++) {
+    newRow.appendChild(col.cloneNode(true));
+  }
+  tableBody.appendChild(newRow);
+
+  if (tableBody.children.length === maxVal) {
+    appendRow.disabled = true;
+  } else {
+    removeRow.disabled = false;
+  }
+});
+
+removeRow.addEventListener('click', () => {
+  tableBody.removeChild(tableBody.lastElementChild);
```

Comment:
try to find tableBody every time when you click on button.  It should help to fix tests.